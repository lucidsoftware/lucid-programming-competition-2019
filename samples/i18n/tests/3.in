50
uttsfvmvtcmestxcn
oevp
zbosuhrdkzfm
mbxzvl
chxupojhabtscdc
vbdo
hmpayjp
schbqjjks
argxfbjtntdv
dwwnxgwgbtah
ghtcjslfryryxkaiwhhd
weoxjwpel
febbnbcfhwbfubmsjd
afhouoaclzixgksy
dfladq
thykrmtjubl
rwu
opet
ddmlpefxgodxi
btucsuuajnkypfxn
pxdapmfxt
tgzk
lltixbammzpvmxtryruk
stcqpepweteuablgax
pvsujkfgwovp
etznvqgmitclexevawbg
zlmglktbtojf
kyshlnagf
fl
knrmhk
nuhyzruxk
qcnfledumowz
whidaxbqeezs
zsqwxwrc
wvszqk
erbwstnjonmudca
xhm
oqkpixwabiwnov
lanmt
yutx
trfcm
utvdubtxmljilupyx
wwfixelpoafqouqywtj
kh
yiapedvptkrvbd
sltip
gnpepfhidskgwzlntdt
etrdht
cv
cgcawm
