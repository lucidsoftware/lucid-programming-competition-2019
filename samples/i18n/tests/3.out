u15n
o2p
z10m
m4l
c13c
v2o
h5p
s7s
a10v
d10h
g18d
w7l
f16d
a14y
d4q
t9l
r1u
o2t
d11i
b14n
p7t
t2k
l18k
s16x
p10p
e18g
z10f
k7f
fl
k4k
n7k
q10z
w10s
z6c
w4k
e13a
x1m
o12v
l3t
y2x
t3m
u15x
w17j
kh
y12d
s3p
g17t
e4t
cv
c4m
